dataverse-webapp/src/main/webapp/selectGuestbook.xhtml

```diff
@@ -0,0 +1,143 @@
+        <ui:param name="dataset" value="#{selectGuestbookPage.dataset}"/>
+        <ui:param name="showMessagePanel" value="#{true}"/>
+        <ui:define name="body">
+            <f:metadata>
+                <f:viewParam name="datasetId" value="#{selectGuestbookPage.datasetId}"/>
+                <f:viewParam name="persistentId" value="#{selectGuestbookPage.persistentId}"/>
+                <f:viewAction action="#{selectGuestbookPage.init}" rendered="true"/>
+                <f:viewAction action="#{dataverseHeaderFragment.initBreadcrumbs(selectGuestbookPage.dataset)}"/>
+            </f:metadata>
+            <h:form id="selectGuestbookForm">
+                <!-- Edit editMode -->
+                <div class="button-block form-top">
+                    <p:commandButton id="saveTop" value="#{bundle.saveChanges}"
+                                     onclick="checkNewlyRestricted();PF('lockSelectGuestbookForm').show();"  />
+                    <p:commandButton id="cancelTop" value="#{bundle.cancel}"
+                                     action="#{selectGuestbookPage.cancel}" />
+                </div>
+                <!-- END Header / Button Panel -->
+
+                <!-- Tabs -->
+                <div class="text-right margin-bottom"
+                     jsf:rendered="#{dataverseSession.user.authenticated and !widgetWrapper.widgetView
+                        and permissionsWrapper.canIssueUpdateDatasetCommand(dataset)}">
+                </div>
+                <div class="panel-group">
+                    <div class="panel panel-default" jsf:rendered="#{!settingsWrapper.rsyncOnly}">
+                        <div id="panelCollapseGuestbook" class="collapse in">
+                            <div class="panel-body">
+                                <div class="form-group">
+                                    <label for="metadata_Guestbook" class="col-sm-3 control-label">
+                                        #{bundle['file.dataFilesTab.terms.list.guestbook']}
+                                        <span class="glyphicon glyphicon-question-sign tooltip-icon"
+                                              data-toggle="tooltip" data-placement="auto right"
+                                              data-original-title="#{bundle['file.dataFilesTab.terms.list.guestbook.title']}"></span>
+                                    </label>
+                                    <div class="col-sm-9">
+                                        <!-- Preview Guestbook from selectGuestbook page -->
+                                        <ui:fragment rendered="true">
+                                            <p class="help-block">#{bundle['file.dataFilesTab.terms.list.guestbook.select.tip']}</p>
+
+                                            <ui:fragment rendered="#{empty selectGuestbookPage.dataset.dataverseContext.availableGuestbooks}">
```

Comment:
Imo it would be easier to understand if we had in selectGuesbookPage field `List<Guestbook> SelectGuestbookPage.availableGuestbooks` initialized on init and used here